src/Template.ts

```diff
@@ -117,12 +117,14 @@ export function create(data: any, context: any, codeFormatter: CodeFormatter): T
 
     handlebarInstance.registerHelper('consumes', function (this: any, type, options: HelperOptions) {
         if (!context.spec.consumers) {
-            return '';
+            return options.inverse(this);
```

Comment:
What does this do ?

Reply:
It enables us to use {{else}}

https://handlebarsjs.com/guide/block-helpers.html#conditionals

Reply:
Nice! Didn't know that one !